versioning/IncrementalVersioning.js

```diff
@@ -0,0 +1,23 @@
+import { BaseVersioning } from "./BaseVersioning";
+
+export class IncrementalVersioning extends BaseVersioning {
+  constructor(releaseHistory) {
+    super(releaseHistory, ([v1], [v2]) => Number(v2) - Number(v1));
+  }
+
+  checkIsMandatory(runtimeVersion) {
+    if (this.sortedMandatoryReleaseHistory.length === 0) {
+      return false;
+    }
+
+    // When latest mandatory version >= current version -> mandatory
+    const [latestMandatoryVersion, _] = this.sortedMandatoryReleaseHistory[0];
+    return Number(latestMandatoryVersion) > Number(runtimeVersion);
```

Comment:
@floydkim 
I think changing the comment is better because when `current version = mandatory version`, it is not necessary to update mandatorily. How do you think about this?
[4484ab8](https://github.com/Soomgo-Mobile/react-native-code-push/pull/27/commits/4484ab8655017238511d07fc270eed973a1ee02f)

Reply:
@changwoolab
Changing the comment is also a good idea.